src/js/components/Tabs/Tabs.js

```diff
@@ -107,6 +108,7 @@ const Tabs = forwardRef(
           wrap
           background={theme.tabs.header.background}
           gap={theme.tabs.gap}
+          margin={headerMargin}
```

Comment:
not sure if we would want this to be a `prop` could we maybe do something in the theme similar to `theme.tabs.header.margin`

Reply:
@ericsoderberghp curious to hear your thoughts around adding a prop vs theme in my opinion if we can achieve this behavior in theme that would be the best place.

Reply:
I think we should allow it to be themed but the prop would be helpful for individually aligning each tabs group separately.

Reply:
@lleBit good point to individually align. In that case this looks good I would just see if you can add a test reflecting this new prop

Reply:
@britt6612 - Can we do both options?

Also, I'm not sure the best way to test this as it's just a margin control on a component, but I'll take a look